PlatformIO/src/StateMachine.hpp

```diff
@@ -68,14 +68,27 @@ class Idle : public StateMachine
     if (xSemaphoreTake(wbSemaphore, (TickType_t)10000) == pdTRUE) {
       device->updateColors(COLORS_IDLE);
     }
-    xEventGroupSetBits(audioGroup, STREAM);
-    xSemaphoreGive(wbSemaphore);
-    initHeader(device->readSize, device->width, device->rate);
+    // start streaming audio to the remote endpoint for hotword detection
```

Comment:
Sorry, already implement I see. Same issue as in HotwordDetected, only the GroupSetBits should be in the if statement.